.gitignore

```diff
@@ -66,3 +66,4 @@ src/packages
 artifacts
 packages
 /Tools
+.config/
```

Comment:
Can you describe what the target for this?

Reply:
I removed this change - nothing is outputing to .config anymore.